javascript-loops-practice/add-suffix-to-all.js

```diff
@@ -0,0 +1,7 @@
+/* exported addSuffixToAll */
+function addSuffixToAll(words, suffix) {
+  var array = [];
+  for (var i = 0; i < words.length; i++) {
+    array.push(words[i] += suffix);
```

Comment:
Oh no, this is bad news because you are changing the original array. You want to use the values of the array given as an argument, but you never want to change the original values.